assets/models/organelles/Nucleus.tscn

```diff
@@ -1,69 +1,27 @@
-[gd_scene load_steps=13 format=3 uid="uid://fut6yvqxhf25"]
+[gd_scene load_steps=6 format=3 uid="uid://fut6yvqxhf25"]
 
-[ext_resource type="ArrayMesh" uid="uid://b6g1xduahoudr" path="res://assets/models/organelles/Nucleus.mesh" id="1"]
-[ext_resource type="Shader" path="res://shaders/OrganelleTransparent.gdshader" id="2"]
-[ext_resource type="Script" path="res://src/microbe_stage/OrganelleMeshWithChildren.cs" id="3"]
-[ext_resource type="ArrayMesh" uid="uid://bvwcnhf6ucku4" path="res://assets/models/organelles/Golgi.mesh" id="4"]
-[ext_resource type="Texture2D" uid="uid://cm6qpo6rqrnjw" path="res://assets/textures/ER.png" id="5"]
-[ext_resource type="Texture2D" uid="uid://vslrd1mpxtgj" path="res://assets/textures/GolgiApparatus.png" id="6"]
-[ext_resource type="ArrayMesh" uid="uid://dcpkvaqyp563n" path="res://assets/models/organelles/ER.mesh" id="7"]
-[ext_resource type="Texture2D" uid="uid://1bv1v578jkl4" path="res://assets/textures/nucleus.png" id="8"]
-[ext_resource type="Texture2D" uid="uid://baxuoyeo83r2u" path="res://assets/textures/dissolve_noise.tres" id="9"]
+[ext_resource type="Shader" path="res://shaders/OrganelleTransparent.gdshader" id="1_bheag"]
+[ext_resource type="Texture2D" uid="uid://baxuoyeo83r2u" path="res://assets/textures/dissolve_noise.tres" id="2_tdl0s"]
+[ext_resource type="Texture2D" uid="uid://bh1e6qrcmeoux" path="res://assets/textures/Nucleus.png" id="3_1w7t3"]
+[ext_resource type="ArrayMesh" uid="uid://buw78wgilybsy" path="res://assets/models/organelles/Nucleus.mesh" id="4_tmwr2"]
 
-[sub_resource type="ShaderMaterial" id="1"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_ngmrq"]
 resource_local_to_scene = true
-render_priority = 0
-shader = ExtResource("2")
+render_priority = 1
```

Comment:
Is this required? This may cause unintended rendering on top of organelles that are in front o the nucleus. The reason this was used before is that the nucleus was multiple parts and it was required to get the draw order for them correct.

Reply:
I was using it while trying out different ways to do the nucleus. It's unnecessary now.

Reply:
Alright so it can be removed, I'll wait for your commit changing that.